as-vending/functions/output.go

```diff
@@ -94,22 +98,22 @@ func (vendingState *VendingState) HandleMqttDeviceReading(edgexcontext *appconte
 						// POST the deltaLedger json string to the ledger endpoint
 						outputBytes, err := json.Marshal(deltaLedger)
 						if err != nil {
-							edgexcontext.LoggingClient.Error("HandleMqttDeviceReading failed to marshal deltaLedger")
+							lc.Error("HandleMqttDeviceReading failed to marshal deltaLedger")
```

Comment:
it is probably beneficial for trouble shooting purpose to include error in the logging message:
```suggestion
							lc.Errorf("HandleMqttDeviceReading failed to marshal deltaLedger: %v", err)
```